install/installer/pkg/components/auth/config_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package auth
+
+import "testing"
+
+func TestCookieNameFromDomain(t *testing.T) {
```

Comment:
Largely generated by ChatGPT, with a prompt to take the JS implementation of this and convert, including write table tests